### ISSUE TITLE
Source code

### ISSUE THREAD

**Issue by theopensour** [cclhk/Invid#2]
I know you said this is dead but why not release the source code of this project so someone can fork it and  continue updating it

**Comment by Phantop** [comment]
Yeah, bumping this as well. The Invidious project is under active development still and under secure management: https://github.com/iv-org/invidious/issues/1411.

This is currently the only way to get >720p30 video with Invidious on iOS and it would be great if the source for it could be released, especially as Invidious (and alternatives using the same API such as CloudTube) has a long future ahead of it.
